calico-cloud/get-started/system-requirements.mdx

```diff
@@ -1,62 +1,50 @@
 
-Verify that your cluster meets these requirements.
+## Kubernetes distributions and CNIs
 
-| Required                    | Supported                                                    |
-| --------------------------- | ------------------------------------------------------------ |
-| **Platforms**               | All of the Calico Open Source [Kubernetes deployments](https://docs.projectcalico.org/getting-started/kubernetes/), except: <br/><br/>- OpenShift<br />- IBM Cloud Kubernetes Service (IKS)<br/>- K3s <br />- Most clusters with Windows nodes. (We support Windows nodes on AKS.)<br />- RKE |
-| **Architecture**            | Only AMD64                                                   |
-| **Kubernetes version**      | - Minimum: v1.25<br/>- Maximum: v1.29<br /><br />If you have a later version than the maximum, contact [Support](https://support.tigera.io/). |
-| **CNIs**                    | - Calico CNI v3.20 to v3.27<br />- Amazon VPC CNI<br/>- Azure CNI<br/>- GKE CNI |
-| **Browsers for Manager UI** | {{prodname}} supports the latest version and "latest minus one" versions of the following browsers:<br/><br/>- Chrome<br/>- Safari<br/>- Firefox |
+Calico Cloud works with Kubernetes on self-provisioned infrastructure and on managed Kubernetes distributions.
+To use Calico Cloud for both networking and network policy, your cluster must have Calico Open Source installed before you connect to Calico Cloud.
+For most managed distributions, you can use the provider's CNI for networking and use Calico Cloud for network policy.
 
-### Step 2: Verify cluster is not managed by a Kubernetes reconciler
+| Distribution | Supported CNIs |
+| --- | --- |
+| Kubernetes on self-provisioned infrastructure | - Calico Open Source 3.26 and 3.27 |
+| Amazon Elastic Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Amazon VPC CNI |
+| Azure Kubernetes Service | - Calico Open Source 3.26 and 3.27<br/> - Azure CNI |
+| Google Kubernetes Engine | - Calico Open Source 3.26 and 3.27<br/> - GKE CNI |
+| Rancher Kubernetes Engine 2 | - Calico Open Source 3.26 and 3.27 |
 
-{{prodname}} cannot be installed on clusters that are managed by any kind of Kubernetes reconciler (for example, Addon-manager). To verify, look for an annotation called `addonmanager.kubernetes.io/mode` on either of the following resources. (The resources may not exist).
 
-- `tigera-operator` deployment in the `tigera-operator` namespace (Installation on AKS may still work if the mode is Reconcile, see [AKS](aks.mdx#verify-addonmanager-is-compatible) for an additional check.)
-- `calico-node` daemonset in the `kube-system` namespace
+## Kubernetes versions
 
-If the following command finds addonmanager on either of the resources, then Addon-manager is being used. Find a different cluster to use.
+Your Kubernetes distribution must be based on one of the following Kubernetes versions:
+* Kubernetes 1.29
+* Kubernetes 1.28
+* Kubernetes 1.27
+* Kubernetes 1.26
```

Comment:
```suggestion
```

Reply:
Since our k8s support is n-2, this is due to the k8s libraries that we use. We usually use an n-1 version of those libraries in our product and per k8s official support (of +- 1) can only call support for n-2.

Reply:
These versions with the v20.0.0 release need to be updated to 1.27-1.30

Reply:
N-2 would put us supporting 1.29-1.31, no? https://kubernetes.io/releases/
@tmjd Does that sound right?

Reply:
No that does not sound right. Enterprise v3.20.0-EP1 supports a max of k8s 1.30 so that is the maximum version CC will support at this time (with v20.0.0)